src/System.Private.Xml/tests/Xslt/XslTransformApi/CXslTransform.cs

```diff
@@ -2928,6 +2747,13 @@ public void var4(InputType inputType, ReaderType readerType, TransformType trans
         }
     }
 
+    internal sealed class AllowDefaultResolverContext : IDisposable
```

Comment:
Per PR #14065 corefx by default disables the default resolver (security by default) in light of that I think it is fine for this type to always ensure that it is disabled when disposed - this way most of the tests can assume the default behavior.

Good point regarding tests running in parallel, what is the xunit attribute for that? (Looking meanwhile...)